src/components/Home-Page/JobsList/job.tsx

```diff
@@ -1,53 +1,33 @@
 import * as React from 'react';
 import Button from '@material-ui/core/Button';
-import Typography from '@material-ui/core/Typography';
 import VisibilityOutlinedIcon from '@material-ui/icons/VisibilityOutlined';
-import StarBorderOutlinedIcon from '@material-ui/icons/StarBorderOutlined';
-import FullStarIcon from '@material-ui/icons/Star';
-import LocationOnOutlinedIcon from '@material-ui/icons/LocationOnOutlined';
-import styles from './jobsListStyle';
-
+import { WithStyles, withStyles } from '@material-ui/core/styles';
+import styles from '../../common/Job/jobStyles';
```

Comment:
`TODO: @common`, Not required by you ^^.  What do you think about it? To make `imports` cleaner? 😃 

Result: No: `../../../../` ladders 🧗

Reply:
i dont see a `Webpack` file in the project now, so we cant do that for now